server/app/views/questiontypes/NameQuestionRenderer.java

```diff
@@ -77,6 +81,22 @@ protected DivTag renderInputTags(
                 validationErrors.getOrDefault(nameQuestion.getLastNamePath(), ImmutableSet.of()))
             .addReferenceClass(ReferenceClasses.NAME_LAST);
 
+    SelectWithLabel nameSuffixField =
+        new SelectWithLabel()
+            .addReferenceClass("cf-dropdown-question")
```

Comment:
(blocking) This should be added to the ReferenceClasses.java file

Reply:
replace it with `ReferenceClasses.NAME_SUFFIX` instead. thanks!